t/complex/physics.xml

```diff
@@ -1117,10 +1117,7 @@
                   <XMApp>
                     <XMTok name="rightarrow" role="ARROW">→</XMTok>
                     <XMTok meaning="absent"/>
-                    <XMDual>
-                      <XMTok meaning="gradient"/>
-                      <XMTok name="nabla" role="OPERATOR">∇</XMTok>
-                    </XMDual>
+                    <XMTok font="italic" meaning="gradient" name="nabla" role="OPERATOR">∇</XMTok>
```

Comment:
I noticed that `font="italic"` gets added to the tokens when merged, I was wondering what causes that and how to avoid it - since I am pretty sure we don't want it added for the cases in the tests.

Reply:
Ok, figured it out, simply do not overwrite the presentation XMTok's attributes with attributes of the parent XMDual, in the cases where both have the same attribute. With a special exception where `xml:id` gets overwritten to be the XMDual id, in case it was being referenced from somewhere else.